src/bundle/Resources/views/content/content_view/content_fields.html.twig

```diff
@@ -0,0 +1,16 @@
+{% set default_fields_group_template = '@EzPlatformAdminUi/content/content_view/fields.html.twig'%}
+{% set fields_group_template = fields_group_template is defined ? fields_group_template : default_fields_group_template %}
```

Comment:
Twig has the null-coalescing operator. I think we can change this to `{% set fields_group_template = fields_group_template ?? default_fields_group_template %}